u8timeseries/utils/timeseries_generation.py

```diff
@@ -0,0 +1,137 @@
+import numpy as np
+import pandas as pd
+import math
+from pandas.tseries.offsets import DateOffset
+from pandas.tseries.frequencies import to_offset
+from pandas.tseries.holiday import USFederalHolidayCalendar
+
+from u8timeseries.timeseries import TimeSeries
+
+
+
+def constant_timeseries(value: float = 0, length: int = 10, offset : str = 'D',
```

Comment:
+1 for type hints and docstrings. 

small nitpick: in offset there is a redundant space before : ;).